tests/python/mpi_test.py

```diff
@@ -0,0 +1,44 @@
+# run it with
+# mpirun -n 2 python mpi_test.py
+
+import torch
+import nvfuser
+from nvfuser import FusionDefinition, DataType
+
+import os
+
+rank = int(os.environ["OMPI_COMM_WORLD_LOCAL_RANK"])
+
+torch.cuda.set_device(rank)
+
+# Inputs
+inputs = [
+    torch.randn(2, 4, device="cuda")[rank : rank + 1, ...],
+]
+
+
+# dynamic shape isn't supported;
+# scalar inputs isn't supported;
+class MultiDeviceModel(FusionDefinition):
+    def definition(self):
+        self.t0 = self.define_tensor((2, 4), (False, False), dtype=DataType.Float)
+        self.t1 = self.ops.relu(self.t0)
+        self.t2 = self.ops.add(self.t1, self.t1)
+        self.add_output(self.t2)
+
+    def schedule(self):
```

Comment:
Can you remind me where `schedule` gets used? I'm trying to understand why TensorView isn't available during `definition` but is during `schedule`.

Reply:
https://github.com/NVIDIA/Fuser/blob/9eba1ad122550bf598f4a8d453d003cea989e4fe/nvfuser/__init__.py#L141

Thanks for the pointer, @jjsjann123!